setup.py

```diff
@@ -1,20 +1,18 @@
-try:
-    from setuptools import setup
-except:
-    from distutils.core import setup
+from setuptools import setup
+
+requirements = ["PyYAML>=4.2", "boto3", "clrypt"]
```

Comment:
Best to indicate recent versions of boto3 and clrypt as minimums as well.

Reply:
Added boto to current color version. With the way we do the internal reqs I don't think I have specify clrypt